tests/foreman/api/test_provisioning.py

```diff
@@ -48,12 +49,12 @@ def test_rhel_pxe_provisioning(
         2. Satellite is able to run REX job on the host
         3. Host is registered to Satellite and subscription status is 'Success'
     """
-    host_mac_addr = provisioning_host._broker_args['provisioning_nic_mac_addr']
+    host_mac_addr = provisioning_host.prov_host._broker_args['provisioning_nic_mac_addr']
     sat = module_provisioning_sat.sat
 
     host = sat.api.Host(
         hostgroup=module_provisioning_rhel_content.hostgroup,
-        organization=module_org_with_manifest,
+        organization=module_org,
```

Comment:
It seems `module_sca_manifest_org` is already getting created with `module_provisioning_rhel_content`, so not sure if creating a new module_org would work, could you check?

Reply:
Or use `module_sca_manifest_org`

Reply:
Thanks @Gauravtalreja1 @Griffin-Sullivan , totally missed this

Reply:
@shubhamsg199 Could you include latest test results to this PR?